easybuild/easyconfigs/f/FLANN/cmake-flann-1.8.4.patch

```diff
@@ -0,0 +1,23 @@
+diff -Nru flann-1.8.4.orig/src/cpp/CMakeLists.txt flann-1.8.4/src/cpp/CMakeLists.txt
```

Comment:
@sassy-crick 
please rename as shown above
please also add a short description (preferentially with a link to the origin) and the author to the top of the patch.

Reply:
Thanks for showing me the git patch command. That was new to me and I spent quite some time to get it working with patch. Something learned! :-)
I have also added the URL of the site where I found that.

Reply:
@sassy-crick please also add the author of the patch? (you :)

Reply:
Done. Same as with the other patch.